demystifying_containers/demystifying_containers.md

```diff
@@ -0,0 +1,165 @@
+
+### Quiz: How containers work
+
+Which of the following describes what a container image is?
+
+[(X)] A set of instructions that describes how a container will be built.
+[( )] A small package of software that has everything it needs to run in any environment.
+[( )] A digital copy of a computer that runs on a server. 
+***
+<div class = "answer">
+
+Unlike a container, which is a lightweight bundle of software with all of the code and dependencies required to run in any environment, a container image is the set of instructions that describes how that container will be built. A digital copy of a computer that runs on a server is a virtual machine.  
+
+</div>
+***
+
+## Container technology
+
+There are a variety of choices when it comes to containerization technology, but the most popular is **Docker**. [Docker's website](https://www.docker.com/) has many resources and articles to help you get started, as well as a [registry for container images, called DockerHub](https://hub.docker.com/), where you can store your own container images and make use of images already out there. 
+
+However there are other providers that you might come across, that are very similar to Docker but might have subtle differences that may make them better for specific workflows (don't worry about those differences right now though). Here are just a few:
+
+- [Podman](https://podman.io/) is a container service developed by [RedHat](https://www.redhat.com/en). It is often used in conjunction with [Buildah](https://buildah.io/).
+- [Linux Daemon, or LXD](https://linuxcontainers.org/)
+- [rkt](https://github.com/rkt/rkt), pronounced like "rocket", is a free container engine that is available on GitHub (although the project has ended and is no longer in active development, you can still use it).
+
+You can also run containers on cloud platforms like [Amazon Web Services, or AWS](https://aws.amazon.com/) and [Google Cloud Platform, or GCP](https://cloud.google.com/) using various orchestration (or automation of the operational tasks that are involved in deploying and managing containers) and computing tools. You can check out [containers in AWS](https://aws.amazon.com/containers/services/) and [containers in GCP](https://cloud.google.com/containers) for more information. 
+
+### Quiz: Container technology
+
+True or false: A Docker container is the only kind of container.
+
+[( )] True
+[(X)] False 
+
+***
+<div class = "answer">
+
+While Docker is the most popular container platform, and has many helpful resources to help you get started, it is not the only option. 
+
+</div>
+***
+
+## Additional Resources
+
+- [Another brief description of what a container is from the Docker website](https://www.docker.com/resources/what-container/)
+- [Article with more information about containers vs. virtual machines](https://www.atlassian.com/microservices/cloud-computing/containers-vs-vms) 
+
```

Comment:
I'm wondering if the "containers in the cloud" stuff you mention at the end of the last section maybe belongs here instead of there? I could see it either way.

Reply:
I've re-worked this section, so I hope it makes more sense now (if it doesn't we can totally revisit)